docs/user/user_registration.md

```diff
@@ -1,16 +1,54 @@
 # User Registration
 
 Before you can begin to manage your projects and access your data within the data-manager, 
-you need to create an account for the platform. For this you need to perform the following steps.
+you need to create an account for the platform. We currently support 2 modes of registration:
 
-1. [Register an account within the data manager](#register-account)
-2. [Validate the created account via the validation link you received in your email](#validate-the-registered-account)
-3. [Login into the data manager platform](#login-to-the-data-manager)
+1. [Register an account via orcid](#register-an-account-via-orcid)
+2. [Register an account via email](#register-an-account-via-email)
 
-## Register Account
+## Register an account via orcid
 
-From the login page you can navigate to the registration view via the "registration" button on the top right 
-or the registration link on the bottom of the login
+### Login to orcid
+
+We recommend to register to the data manager via your existing [orcid](https://orcid.org) account.
+  
+To do so press on the "login with ORCiD" registration card on the bottom of the login page. 
+
+![login_page_registration](images/user_registration/login_page.png)
+
+This will redirect you to the orcid login page 
+
+![orcid_login](images/user_registration/orcid/orcid_login.png)
+
+Login to your orcid account and grant the data manager application to access your publicly available orcid by pressing the "authorize access" button. 
+
+![orcid_registration_authorize_access.png](images/user_registration/orcid/orcid_registration_authorize_access.png).
+
+Should you not have an orcid account press the "register_now" link 
+within the orcid login and follow the provided steps to register a new orcid account.
+
+### Finalize data manager account
+
+If done correctly you will be redirected to the data manager application to finalize your account registration by providing us with additional user information.
+![orcid_registration_data_manager_registration.png](images/user_registration/orcid/orcid_registration_data_manager_registration.png)
+
+This additional information consists of
+
+1. Your full name consisting of your first name and surname
+2. A valid email address of choice which will be associated with your data manager account
+3. A unique username which identifies you within the data manager platform
+
+!!! info "Username"
+    This username is visible to other users within the platform
+
+Press the register button to register an account within the data manager application and navigate to the login page.
+Finally, press on the "login with ORCiD" registration card on the bottom of the login page to login into the data manager application via your orcid credentials. 
```

Comment:
The wrong image is displayed here, there is some password input field instead of the "login with orcid"

Reply:
Nice Spot!